api/api.go

```diff
@@ -128,7 +128,10 @@ func StartApi() error {
 	srv := handler.NewDefaultServer(graphql.NewSchema(entClient))
 	srv.Use(entgql.Transactioner{TxOpener: entClient})
 
-	app.GET("/", playgroundHandler())
+	app.Static("/assets", "./assets")
+	app.StaticFile("/", "./index.html")
```

Comment:
I believe this isn't a catch all - if I refresh a page I just get a 404

Reply:
will be tackled by nint